src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -143,18 +153,15 @@ internal SourceText ReadFileContent(CommandLineSourceFile file, IList<Diagnostic
         /// <param name="diagnostics">Storage for diagnostics.</param>
         /// <param name="normalizedFilePath">If given <paramref name="file"/> opens successfully, set to normalized absolute path of the file, null otherwise.</param>
         /// <returns>File content or null on failure.</returns>
-        internal SourceText ReadFileContent(CommandLineSourceFile file, IList<DiagnosticInfo> diagnostics, out string normalizedFilePath)
+        internal SourceText TryReadFileContent(CommandLineSourceFile file, IList<DiagnosticInfo> diagnostics, out string normalizedFilePath)
```

Comment:
This was for consistency re: feeback to make analogous TryReadEmbeddedFileContent express it can fail and return null. But I neglected the overload without the normalizedFilePath. I will rename that too.